documentation/previous-versions-quickstart.md

```diff
@@ -0,0 +1,531 @@
+# Azure SDK for Go - Previous Versions
```

Comment:
this is the document for track 1 and previously on readme page (it is very long, so moving to here to provide clear distinction between track 1 and track 2)